tests/python/unittest/test_numpy_op.py

```diff
@@ -1647,6 +1647,50 @@ def hybrid_forward(self, F, a):
                         assert_almost_equal(mx_out.asnumpy(), np_out, rtol=1e-3, atol=1e-5)
 
 
+@with_seed()
+@use_np
+def test_np_shifts():
+    class TestShift(HybridBlock):
+            def __init__(self, func):
+                super(TestShift, self).__init__()
+                self._func = func
+
+            def hybrid_forward(self, F, a, b):
+                return getattr(F.np, self._func)(a, b)
+
+    shapes = [
+        ((), ()),
+        ((), (2,)),
+        ((3,), ()),
+        ((3, 4), (3, 4)),
+        ((3, 4, 5, 6), (6,)),
+        ((2, 0, 3), (3,)),
+        ((2, 1, 3), (2, 3))
+    ]
+    dtypes = ['uint8', 'int8', 'int32', 'int64']
```

Comment:
Please include the test of floating data types. Use assert_exception to catch the exception and show the proper error messages.